src/zkevm_specs/evm/execution/comparator.py

```diff
@@ -0,0 +1,51 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition
+from ..opcode import Opcode
+
+
+def cmp(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    is_eq, is_gt = instruction.pair_select(opcode, Opcode.EQ, Opcode.GT)
```

Comment:
I think `pair_select` is implemented like https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/5710e30c887d90e4b5e93bf8e332028dee2ac223/zkevm-circuits/src/evm_circuit/util/math_gadget/pair_select.rs#L24-L39, where the `opcode` must be either `Opcode.EQ` or `Opcode.GT` (but here it could also be `Opcode.LT`).

Here I think we could just say
```suggestion
    is_eq = instruction.is_equal(opcode, Opcode.EQ)
    is_gt = instruction.is_equal(opcode, Opcode.GT)
```

So it might look more similar to the circuit implementation here https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/5710e30c887d90e4b5e93bf8e332028dee2ac223/zkevm-circuits/src/evm_circuit/execution/comparator.rs#L27-L28.